src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/config/ImporterConfiguration.kt

```diff
@@ -19,6 +17,7 @@ class ImporterConfiguration {
     fun clock() = Clock.systemDefaultZone()
 
     @Bean
-    @Qualifier(value ="spreadsheet-template")
-    fun file(@Value("\${export-files.template}") location: String) : File = resourceLoader.getResource(location).file
+    fun jcpTemplateProvider(@Value("\${export-files.template}") templateFileLocation: String): JCPTemplateProvider {
+        return JCPTemplateProvider { resourceLoader.getResource(templateFileLocation).inputStream }
```

Comment:
Loading this with the ResourceLoader as an InputStream resolves the issue with the resource being contained in the resources folder in the JAR file.